sdk/typescript/graphql/client.ts

```diff
@@ -14,6 +35,7 @@ class CustomSetter {
 
 export function createGQLClient(port: number, token: string): GraphQLClient {
   const client = new GraphQLClient(`http://127.0.0.1:${port}/query`, {
+    fetch: createFetchWithTimeout(1000 * 60 * 30), // 30minutes timeout
```

Comment:
There's realistically no expected timeout anywhere; if the user is performing an operation that takes several hours we don't want to timeout.

If we are forced to set this to some value or else it will default to a low value, then I'd just set it as high as it can be (or like 1 week, something very very long).

cc @helderco @jedevc @vito in case I'm forgetting something where we have an expected operation timeout in our stack somewhere

Reply:
I extended it to a week, because it's quite obscur what's the timeout config in the `graphql-request` client.

Reply:
It seems node-fetch has a 5minute timeout by default now (based on https://github.com/nodejs/undici/issues/1373) which is not great for our case.

Reply:
@TomChv does this change override that fetch in undici timeout, or do you need to re-import it and override it, etc?

Reply:
Yeah this override it